common/remote_util.h

```diff
@@ -113,9 +118,10 @@ class RemoteUtil {
   ProcessFactory* const process_factory_;
   const bool forward_output_to_log_;
 
-  SdkUtil sdk_util_;
-  std::string gamelet_ip_;
-  int ssh_port_ = 0;
+  std::string scp_command_ = "scp.exe";
```

Comment:
Again, can you drop the `.exe` suffix to make it easier to port to Linux?

Reply:
Done.